src/transform/ir_proto_to_datalog/logging/context_logger.h

```diff
@@ -0,0 +1,62 @@
+#ifndef SRC_TRANSFORM_IR_PROTO_TO_DATALOG_LOGGING_CONTEXT_LOGGER_H_
+#define SRC_TRANSFORM_IR_PROTO_TO_DATALOG_LOGGING_CONTEXT_LOGGER_H_
+
+#include "absl/strings/string_view.h"
```

Comment:
Can't we use https://github.com/google/glog?

Reply:
Thank you for this pointer, I will use that library.